### PR TITLE
IBX-665: Bumped Symfony dependency to 5.3

### PR DESCRIPTION
**Pull request by webhdx**
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-665](https://issues.ibexa.co/browse/IBX-665)
| **Type**                                   |improvement
| **Target Ibexa version** | `v3.3`
| **BC breaks**                          | no
| **Doc needed**                       | no

#### Checklist:
- [x] Provided PR description.
- [x] Tested the solution manually.
- [x] Checked that target branch is set correctly.
- [x] Asked for a review (ping `@ibexa/engineering`).

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -41,7 +42,7 @@
     },
     "flex-require-dev": {
         "symfony/debug-pack": "*",
-        "symfony/maker-bundle": "^1.26.1",
+        "symfony/maker-bundle": "^1.0",
```
Comment (Steveb-p) [inline composer.json:45]: Why is this version (and `symfony/flex`) lowered?

Comment (lserwatka) [reply composer.json:45]: Consistency maybe? @webhdx

Comment (webhdx) [reply composer.json:45]: Yes, I aligned it with `symfony/website-skeleton`

Comment (lserwatka) [reply composer.json:45]: It will pick up latest version anyway while we are tagging.

Comment (Steveb-p) [reply composer.json:45]: I'm ok with it, just curious.

**Review by Steveb-p — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

**Review by mnocon — APPROVED** [review]

**Pull request merged**